Homework-05-files-encoding-formats.ipynb

```diff
@@ -46,11 +76,83 @@
+      "CONFERENCE_PAPER\n",
+      "Term Extraction and Automatic Indexing\n",
+      "[{'name': 'Christian Jacquemin', 'uri': None, 'localAuthorityId': None, 'firstName': 'Christian', 'lastName': 'Jacquemin', 'positionInList': 0, 'affiliations': None}, {'name': 'Didier Bourigault', 'uri': None, 'localAuthorityId': None, 'firstName': 'Didier', 'lastName': 'Bourigault', 'positionInList': 1, 'affiliations': None}, {'name': 'Ruslan Mitkov', 'uri': None, 'localAuthorityId': None, 'firstName': 'Ruslan', 'lastName': 'Mitkov', 'positionInList': 2, 'affiliations': None}]\n",
+      "BOOK_SECTION\n",
+      "The Anatomy of a Nanopublication\n",
+      "[{'name': 'Paul Groth', 'uri': None, 'localAuthorityId': None, 'firstName': 'Paul', 'lastName': 'Groth', 'positionInList': 0, 'affiliations': None}, {'name': 'Andrew Gibson', 'uri': None, 'localAuthorityId': None, 'firstName': 'Andrew', 'lastName': 'Gibson', 'positionInList': 1, 'affiliations': None}, {'name': 'Jan Velterop', 'uri': None, 'localAuthorityId': None, 'firstName': 'Jan', 'lastName': 'Velterop', 'positionInList': 2, 'affiliations': None}]\n",
+      "JOURNAL_ARTICLE\n",
+      "add author:Hrry\n",
+      "add title:aodsf\n",
+      "add item type:asd\n"
+     ]
+    },
+    {
+     "ename": "NameError",
+     "evalue": "name 'new_entry' is not defined",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
+      "\u001b[0;32m<ipython-input-76-6322819453e9>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m     22\u001b[0m         \u001b[0mnew_title\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0minput\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"add title:\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     23\u001b[0m         \u001b[0mnew_itemtype\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0minput\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"add item type:\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 24\u001b[0;31m         \u001b[0mf\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mwritelines\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mnew_entry\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
+      "\u001b[0;31mNameError\u001b[0m: name 'new_entry' is not defined"
+     ]
+    }
+   ],
    "source": [
-    "# your code"
+    "import json\n",
+    "\n",
+    "with open(\"/Users/Anna/Documents/python-for-the-humanities/python-for-the-humanities-homework/HW-05-data.json\") as d:\n",
+    "    data=json.load(d)\n",
+    "    get_items=data[\"items\"]\n",
```

Comment:
`get_items` is a bad variable names as it implies an action (getting something). verbs are usually used for function/method names; variables are typically nouns.